server/services/follow-services.js

```diff
@@ -0,0 +1,63 @@
+const Follow = require('../models/Follow.js');
+
+async function getUserFollowing(userId) {
+  const follows = await Follow.find({ user: userId }).populate('following');
+  const followingList = follows.map((follow) => ({
+    username: follow.following.username,
+    fullName: follow.following.fullName,
+    profilePic: follow.following.profilePic,
+    created: follow.created
+  }));
+  return followingList;
+}
+
+async function getUserFollowers(userId) {
+  const follows = await Follow.find({ following: userId }).populate('user');
+  const followerList = follows.map((follow) => ({
+    username: follow.user.username,
+    fullName: follow.user.fullName,
+    profilePic: follow.user.profilePic,
+    created: follow.created
+  }));
+  return followerList;
```

Comment:
technically you could just return the follows.map you don't have to save it in a variable. If you want.

like so:

`async function getUserFollowers(userId) {
  const follows = await Follow.find({ following: userId }).populate('user');
  return follows.map((follow) => ({
    username: follow.user.username,
    fullName: follow.user.fullName,
    profilePic: follow.user.profilePic,
    created: follow.created
  }));
}
`